File: client/src/app/_models/admin/assessmentQBank.ts
import { IAssessmentStandardQ } from "./assessmentStandardQ";

export interface IAssessmentQBank
{
     id: number;
     professionId: number;
     professionName: string;
     assessmentStddQs: IAssessmentStandardQ[];
}
export class AssessmentQBank
{
     id=0;
     professionId=0;
     professionName='';
     assessmentStddQs=null;
}

export interface IAssessmentQBankItem
{
     id: number;
     assessmentQBankId: number;
     assessmentParameter: string;
     qNo: number;
     isStandardQ: boolean;
     question: string;
     maxPoints: number;
}

File: api/Entities/HR/AssessmentQBank.cs
namespace api.Entities.HR
{
    public class AssessmentQBank: BaseEntity
    {
        public int ProfessionId { get; set; }   
        public string ProfessionName { get; set; }
        public ICollection<AssessmentStddQ> AssessmentStddQs { get; set; }
     
    }
}